src/module-1/cards-list-v1/index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="index-style-card.css">
+        <link rel="stylesheet" type="text/css" href="index-cards-list.css">
+        <style>
+            @import url('https://fonts.googleapis.com/css2?family=Barlow:wght@200&display=swap');
+        </style>
+    </head>
+    <body>
+        <div class="catalog">
+            <div class = "product-card">
```

Comment:
There should be no space before and after `=` in attributes declarations.

`class="product-card"`